include/astshim/FrameSet.h

```diff
@@ -409,7 +409,10 @@ friend class Object;
     };
 
     /**
-    Remove a @ref Frame Frame from a @ref FrameSet
+    Remove a @ref Frame Frame from a @ref FrameSet.
```

Comment:
Basically that's the case. I left it in for consistency with the rest of the documentation.

I agree that links such as `@ref Frame "Frames"` could probably be changed to `Frames` with little harm done. However, references to properties and other sorts of anchors really do benefit from having the link text specified, and there are a lot of those. Furthermore, any link that specifies link text needs more than just deleting `@ref`, since doing that would leave the a word followed by the desired link text behind.